src/csm_testing/tests/iuf_apis/apis_session_function.py

```diff
@@ -0,0 +1,188 @@
+
+import sys
+from .apis_activity_functions import count, total_count
+
+def get_sessions(apis, activity):
+    global count , total_count
```

Comment:
`total_count` is not assigned to in this function, so it does not need to be declared here. 
```suggestion
    global count
```